pkg/client/dtclient/settings_client.go

```diff
@@ -841,3 +862,106 @@ func (d *SettingsClient) Delete(ctx context.Context, objectID string) error {
 
 	return nil
 }
+
+func (d *SettingsClient) GetPermission(ctx context.Context, objectID string) (PermissionObject, error) {
+	path, err := getPermissionPathWithID(objectID)
+	if err != nil {
+		return PermissionObject{}, err
+	}
+
+	resp, err := coreapi.AsResponseOrError(d.client.GET(
```

Comment:
Any reason you preferred the deprecated over the non-deprecated methods?

Reply:
To consistently use the same method over the client. In the deprecated call, we're using the new method. Maybe we should change it for all methods in the client in a follow-up?

Reply:
I think we should have consistent usage, and not using deprecated functions. A unification would be appreciated :)